bindata/network/ovn-kubernetes/common/ipsec-host.yaml

```diff
@@ -205,7 +210,8 @@ spec:
           # Check nss database status
           /usr/sbin/ipsec --checknss
           # Start the pluto IKE daemon
-          /usr/libexec/ipsec/pluto --leak-detective --config /etc/ipsec.conf --logfile /var/log/openvswitch/libreswan.log
+          ## we'll no longer start it ourseleves but rely on prerequisit to enable the systemd service
+          #/usr/libexec/ipsec/pluto --leak-detective --config /etc/ipsec.conf --logfile /var/log/openvswitch/libreswan.log
```

Comment:
@yuvalk Just remove the command itself, don't comment it out. Kill the "# start the pluto ike daemon" comment too since it'll get started elsewhere.

But really, how does the IKE daemon get started now? How does the systemd unit get enabled now?

Reply:
will do